### DISCUSSION TITLE
Page not indexing in search

### DISCUSSION THREAD

**Discussion by pyjs-tmdc** [squidfunk/mkdocs-material#7843] [Q&A]
## Issue Description

I’ve encountered an issue where a page I’ve added to the mkdocs.yml file is not being indexed in the search results. Although the page is correctly added to the navigation structure, it doesn’t appear in the search index.

## Possible Cause

This could be due to the same term appearing in multiple other pages, causing search relevance issues or prioritization conflicts. As a result, the desired page may be getting deprioritized or omitted entirely from the search results.

## Link to mkdocs.yml

https://github.com/moderndatacompany/dataos.info/blob/81bd2cd9b75e2d2218127c5bda6bd0a292317fb6/mkdocs.yml#L511

## Link

![image](https://github.com/user-attachments/assets/70dedb28-6614-41bd-90b5-5b68a8224802)

**Comment by kamilkrzyskow** [comment]
Hello @pyjs-tmdc,
the page is inside the [search_index.json](https://dataos.info/search/search_index.json) file, so it is caught during the `search` plugin processing.
I see some error in the browser console when doing a search, `dataproduct` finds some `DataProduct`, so it works partially but not in full.

My suggestion would be to disable as much of extra features / extensions / javascripts etc. your website is using, and see if the search works then. Basically follow the guide on how to create the [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

BTW, why is there a `mkdocs_material.egg-info` directory in the repository root? Seems unnecessary, so perhaps there are some other unnecessary files that create an edge case? Doubt it, but if removing the extras won't work, then this would be the next step in the investigation.

Last idea would be a simple check if the theory about the too many usages of Data Product holds water, and temporarily remove half of the docs and see if the situation improves.
